### ISSUE TITLE
helper functions for [String] and [Text]

### ISSUE THREAD

**Issue by clojj** [chris-martin/loc#5]

**Comment by chris-martin** [comment]
Yes, I think this is a natural request. If you have any code to suggest, I'd be happy to see it. If I recall correctly, I didn't include anything like this because I had some difficulty reasoning about some corner cases at line breaks.
